Sources/CasePaths/Documentation.docc/CaseKeyPath.md

```diff
@@ -14,5 +14,4 @@
 ### Partial case paths
 
 - ``PartialCaseKeyPath``
-- ``Swift/PartialKeyPath/callAsFunction(_:)``
```

Comment:
I don't think this actually exists. Is this old?

Reply:
Looks like it moved to `_AppendKeyPath` to properly work. Maybe doesn't need to be documented though.